### PR TITLE
Add ship name and class to intro screen.

### PR DESCRIPTION
**Pull request by ecraven**
screenshot

### CODE REVIEW

**Comment by nozmajner** [comment]
Nice idea!

**Comment by impaktor** [comment]
Your code doesn't conform to clang thingy.

I've compiled and tested. Works as advertised.

**Comment by nozmajner** [comment]
Erm, on that screenshot, the ship and the text doesn't match.

**Comment by laarmen** [comment]
Soooo... Not as advertised, then :)

**Comment by impaktor** [comment]
> Soooo... Not as advertised, then :)

I'm required to know the name of all the ships in game? What do you ask of me? It didn't crash and that was good enough for me.

**Comment by nozmajner** [comment]
I don know, but didn't spotted it at first.

**Comment by laarmen** [comment]
You didn't know ? All ships, as well as the names of all systems less than
100ly away from Sol.

Nice catch, noz :)

File: src/Intro.h [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Intro : public Cutscene {
 	Intro(Graphics::Renderer *r, int width, int height);
 	~Intro();
 	virtual void Draw(float time);
+	SceneGraph::Model *getCurrentModel() const { return m_model; }
```
Comment (mike-f1) [inline src/Intro.h]: `const` ?

Comment (ecraven) [reply src/Intro.h]: does not modify the object, thus cunst.

**Comment by Keeper1st** [comment]
I found the font size of 24 instead of 30 looked a little better. It would be really slick if the name could be displayed only when the ship is fully zoomed in. (Centering the name would be nice too...)

**Comment by ecraven** [comment]
> I found the font size of 24 instead of 30 looked a little better. It would be really slick if the name could be displayed only when the ship is fully zoomed in. (Centering the name would be nice too...)

Does #4534 work for you?

**Pull request merged**